py/picca/delta_extraction/data.py

```diff
@@ -600,3 +645,65 @@ def save_deltas(self):
             self.rejection_log.add_to_rejection_log(forest, "accepted")
 
         self.rejection_log.save_rejection_log()
+
+    def rename_exposures(self):
+        """In case there are not coadded forest, rename them to make independent the delta 
+        extraction
+        """
+        los_id_list = np.array([forest.los_id for forest in self.forests])
+        unique_los_id_list = np.unique(los_id_list)
+        if unique_los_id_list.size != los_id_list.size:
+            for los_id in unique_los_id_list:
+                for i, forest in enumerate(
+                    np.array(self.forests)[los_id_list == los_id]
+                ):
+                    forest.los_id = f"{los_id}_{i}"
+
+
+    def select_best_exposures(self):
```

Comment:
this is currently called on all analyses by survey.py when computing mean expected flux, I think the decision on if we need to select the best exposure should go there

Reply:
in the normal case, it does not select the best exposure.

Reply:
yes, but I think it would be nicer to have that function actually just do the selection, and only call it from survey.py when we want to select a best exposure...